lib/web_ui/lib/src/engine/pointer_converter.dart

```diff
@@ -138,12 +158,13 @@ class PointerDataConverter {
   }
 
   bool _locationHasChanged(int device, double physicalX, double physicalY) {
-    assert(_pointers.containsKey(device));
-    final _PointerState state = _pointers[device]!;
+    assert(globalPointerState.pointers.containsKey(device));
+    final _PointerDeviceState state = globalPointerState.pointers[device]!;
```

Comment:
I don't mind globalPointerState being a global, but it could be modeled as a property of a PointerDevice object and do:

```dart
_PointerDeviceState state = pointerDevices[deviceId].state!;
```

But since we don't care about anything else of the pointerDevice -> ship it like this!